src/mui/layout/withResources.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
```

Comment:
I was planning to document it for use in custom menu. I guess a selector will do too